src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift

```diff
@@ -48,8 +48,8 @@ class HealthCertificateViewController: UIViewController, UITableViewDataSource,
 		setupViewModel()
 	}
 
-	override func viewDidDisappear(_ animated: Bool) {
-		super.viewDidDisappear(animated)
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
```

Comment:
Good question: Because we want to make sure that the user has seen it. The viewWillAppear life cycle hook is triggered before the display of the screen.